### PR TITLE
Starter Kit Docs

### PR DESCRIPTION
**Pull request by rrecuero**
Updates the docs for starter kit

### CODE REVIEW

File: docs/modules/ROOT/pages/gsnkit.adoc [hunk 1]
```diff
@@ -23,75 +15,97 @@ starter kit you want to use. In this case we'll start our project using the kit
...
+Also, the counter contract is going to naively pay for all the transactions that are submitted.
+Note how the `acceptRelayedCall` determines this by returning 0.
+
+[source,solidity]
+----
+pragma solidity ^0.5.0;
+
+import "tabookey-gasless/contracts/GsnUtils.sol";
+import "tabookey-gasless/contracts/IRelayHub.sol";
+import "tabookey-gasless/contracts/RelayRecipient.sol";
```
Comment (spalladino) [inline docs/modules/ROOT/pages/gsnkit.adoc]: Let's use the openzeppelin contracts in the docs instead. We can update this code as soon as we update the contract in the dapp.

File: docs/modules/ROOT/pages/gsnkit.adoc [hunk 2]
```diff
@@ -23,75 +15,97 @@ starter kit you want to use. In this case we'll start our project using the kit
...
+that are indistinguishable from Web2.0 apps.
+
+This documents assumes familiarity with Gas Station Network. Here are some resources about it:
+
+https://gsn.openzeppelin.com/[Website]
+https://docs.openzeppelin.com/contracts/2.x/gsn[GSN Contracts Overview]
+https://github.com/OpenZeppelin/openzeppelin-gsn-provider[GSN Provider]
+
+### How does it use Web3 with GSN?
+This kit uses Open Zeppelin https://github.com/OpenZeppelin/openzeppelin-network.js[network.js] to create the connection to Web3. Using a couple
```
Comment (spalladino) [inline docs/modules/ROOT/pages/gsnkit.adoc:31]: > Using a couple

I'd add more detail as to which flags these are, so the reader knows where to look

File: docs/modules/ROOT/pages/gsnkit.adoc [hunk 3]
```diff
@@ -23,75 +15,97 @@ starter kit you want to use. In this case we'll start our project using the kit
...
+    setIsDeployed(isDeployed);
+    if (isDeployed) {
+      const funds = await getRecipientFunds(lib, _address);
+      setFunds(funds);
+    }
+  }
+};
+----
+
+You can top your balance by sending funds to your contract using `oz transfer` command.
```
Comment (spalladino) [inline docs/modules/ROOT/pages/gsnkit.adoc]: Watch out! A transfer doesn't cut it. You actually need to call a specific method on the hub itself. You can use `npx oz-gsn fund-recipient --recipient ADDRESS`, or go to the dapp tool to do it.

File: docs/modules/ROOT/pages/quickStart.adoc [hunk 4]
```diff
@@ -1,24 +1,16 @@
...
-
-``
-npm install -g truffle@5.0.2 ganache-cli@6.3.0 @openzeppelin/cli@2.5.1
-``
-
 ## Installation
 
 Ensure you are in a new and empty directory, and run the `unpack` command with the name of the
-starter kit you want to use. In this case we'll start our project using the kit with the name `starter`.
+starter kit you want to use. e.g `starter`
```
Comment (spalladino) [inline docs/modules/ROOT/pages/quickStart.adoc]:
```suggestion
starter kit you want to use, e.g. `starter`.
```

**Review by spalladino — COMMENTED** [review]
Looking much better now! I left a few comments on the gsn kit doc.

File: docs/modules/ROOT/pages/gsnkit.adoc (hunk 2 above)
Comment (rrecuero) [reply docs/modules/ROOT/pages/gsnkit.adoc:31]: That should be in network.js documentation. We can link to it once we have it

File: docs/modules/ROOT/pages/gsnkit.adoc (hunk 1 above)
Comment (rrecuero) [reply docs/modules/ROOT/pages/gsnkit.adoc]: sounds good

File: docs/modules/ROOT/pages/quickStart.adoc (hunk 4 above)
Comment (rrecuero) [reply docs/modules/ROOT/pages/quickStart.adoc]: changed

File: docs/modules/ROOT/pages/gsnkit.adoc (hunk 3 above)
Comment (rrecuero) [reply docs/modules/ROOT/pages/gsnkit.adoc]: changed

**Pull request merged**